src/inmanta/ast/entity.py

```diff
@@ -329,12 +329,17 @@ def get_instance(
 
     def is_subclass(self, subclass_candidate: "Entity", *, strict: bool = True) -> bool:
         """
-        Check if the given subclass_candidate entity is a subclass of this class.
+        Check if self is a subclass off the given subclass_candidate.
         Does not consider entities a subclass of themselves in strict mode (the default).
 
         :param strict: Only return True for entities that are a strict subtype, i.e. not of the same type.
         """
-        return (not strict and subclass_candidate == self) or subclass_candidate.is_parent(self)
+        return (not strict and subclass_candidate == self) or self.is_parent(subclass_candidate)
```

Comment:
This is annoying, `is_parent` semantic is reversed compared to `is_subclass` now

Reply:
annoying, but I'm not going to touch that one.